README.md

```diff
@@ -180,7 +190,7 @@ usage: pwcpp [-h] [-m MODEL] [--version] [--processors PROCESSORS] [-otxt] [-ovt
              [--translate TRANSLATE] [--no_context NO_CONTEXT] [--single_segment SINGLE_SEGMENT] [--print_special PRINT_SPECIAL]
              [--print_progress PRINT_PROGRESS] [--print_realtime PRINT_REALTIME] [--print_timestamps PRINT_TIMESTAMPS]
              [--token_timestamps TOKEN_TIMESTAMPS] [--thold_pt THOLD_PT] [--thold_ptsum THOLD_PTSUM] [--max_len MAX_LEN]
-             [--split_on_word SPLIT_ON_WORD] [--max_tokens MAX_TOKENS] [--speed_up SPEED_UP] [--audio_ctx AUDIO_CTX]
```

Comment:
BTW, could you please make this change in a separate commit and the submodule update in its own commit as well? This will help keep the change history clear in case we ever need to revert back.

I've always wanted to remove the `speed_up` but didn't find time. 
Thanks a lot!

Reply:
How should I do it in separate commit?
I think that once I'll revert this commit it will close the PR

Reply:
In that case, I think just close this PR, pull the latest changes, test if everything is working on Windows with the small fix I added yesterday, 
Then submit a PR to remove the speed_up, then update whisper.cpp and submit another PR. 

Thanks a lot!